plonetheme/barceloneta/theme/less/portlets.plone.less

```diff
@@ -57,6 +57,7 @@
 				position: relative;
 				z-index: 1;
 				color: @plone-link-color-on-grey;
+				&.state-private { color: @plone-default-private-color }
```

Comment:
There are more states than state-private. See: https://github.com/plone/plonetheme.barceloneta/blob/master/plonetheme/barceloneta/theme/less/states.plone.less
Therefore this fix doesn't catch all the other cases.
I propose to use the ``!important`` modifier in states.plone.less.